org.doctales.release-notes.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "org.doctales.release-notes",
+    "description": "DITA-OT plugin for generating release notes from DITA topics.",
+    "keywords": ["Release Notes"],
+    "homepage": "https://github.com/doctales/org.doctales.release-notes/",
+    "vers": "1.1.0",
+    "license": "Apache-2.0",
+    "deps": [
+      {
+        "name": "org.dita.base",
+        "req": ">=2.3"
```

Comment:
We added more strict validation and now all versions need to be valid semver versions. So this must be `>=2.3.0`. Then the validation should go through.